Backpack/src/main/res/values-night/backpack.semantic.color.xml

```diff
@@ -18,76 +18,90 @@
+	<color name="bpkStatusWarningSpot">#ffffb54d</color>
 
-	<color name="bpkStatusDangerFill">#ffcadd</color>
+	<color name="bpkStatusDangerFill">#ffffcadd</color>
 
-	<color name="bpkStatusWarningFill">#fbf1bb</color>
+	<color name="bpkStatusWarningFill">#fffbf1bb</color>
 
-	<color name="bpkSurfaceContrast">#1d1b20</color>
+	<color name="bpkSurfaceContrast">#ff1d1b20</color>
 
-	<color name="bpkSurfaceDefault">#1d1b20</color>
+	<color name="bpkSurfaceDefault">#ff1d1b20</color>
 
-	<color name="bpkSurfaceElevated">#2c2c2e</color>
+	<color name="bpkSurfaceElevated">#ff2c2c2e</color>
 
-	<color name="bpkSurfaceHighlight">#2c2c2e</color>
+	<color name="bpkSurfaceHighlight">#ff2c2c2e</color>
 
-	<color name="bpkTextOnDark">#ffffff</color>
+	<color name="bpkTextOnDark">#ffffffff</color>
 
-	<color name="bpkTextOnLight">#111236</color>
+	<color name="bpkTextOnLight">#ff111236</color>
 
-	<color name="bpkTextLink">#6d9feb</color>
+	<color name="bpkTextLink">#ff6d9feb</color>
 
-	<color name="bpkTextDisabled">#8e8e93</color>
+	<color name="bpkTextDisabled">#ff8e8e93</color>
 
-	<color name="bpkTextPrimary">#ffffff</color>
+	<color name="bpkTextPrimary">#ffffffff</color>
 
-	<color name="bpkTextQuaternary">#8e8e93</color>
+	<color name="bpkTextQuaternary">#ff8e8e93</color>
 
-	<color name="bpkTextError">#d1435b</color>
+	<color name="bpkTextError">#ffd1435b</color>
 
-	<color name="bpkTextTertiary">#8e8e93</color>
+	<color name="bpkTextTertiary">#ff8e8e93</color>
 
-	<color name="bpkTextSecondary">#b2b2bf</color>
+	<color name="bpkTextSecondary">#ffb2b2bf</color>
 
-	<color name="bpkTextPrimaryInverse">#111236</color>
+	<color name="bpkTextPrimaryInverse">#ff111236</color>
 
-	<color name="bpkScrim">#02122c</color>
+	<color name="bpkScrim">#ff02122c</color>
 
-	<color name="bpkBackgroundElevation03">#3a3a3c</color>
+	<color name="bpkBackgroundElevation03">#ff3a3a3c</color>
 
-	<color name="bpkBackgroundElevation02">#2c2c2e</color>
+	<color name="bpkBackgroundElevation02">#ff2c2c2e</color>
 
-	<color name="bpkLine">#48484a</color>
+	<color name="bpkLine">#ff48484a</color>
 
-	<color name="bpkBackgroundElevation01">#1d1b20</color>
+	<color name="bpkBackgroundElevation01">#ff1d1b20</color>
 
-	<color name="bpkBackgroundAlternativeSecondary">#1d1b20</color>
+	<color name="bpkBackgroundAlternativeSecondary">#ff1d1b20</color>
 
-	<color name="bpkBackground">#000000</color>
+	<color name="bpkBackground">#ff000000</color>
 
-	<color name="bpkBackgroundTertiary">#2c2c2e</color>
+	<color name="bpkBackgroundTertiary">#ff2c2c2e</color>
 
-	<color name="bpkBackgroundSecondary">#1d1b20</color>
+	<color name="bpkBackgroundSecondary">#ff1d1b20</color>
 
-	<color name="bpkBackgroundAlternative">#000000</color>
+	<color name="bpkBackgroundAlternative">#ff000000</color>
 
-	<color name="bpkPrimary">#6d9feb</color>
+	<color name="bpkPrimary">#ff6d9feb</color>
+
+	<color name="__privateChipOnDarkNormalBackground">#ff0b121d</color>
```

Comment:
these are the new private tokens. in the view world moving them to a separate file won't help much as nobody accesses them via a file